docs/site/Parsing-requests.md

```diff
@@ -86,28 +86,32 @@ string, see
 and
 [Style Examples](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#style-examples).
 
-At the moment, LoopBack supports object values for parameters in query strings
-with `style: "deepObject"` only. Please note that this style does not preserve
-encoding of primitive types, numbers and booleans are always parsed as strings.
+At the moment, LoopBack supports both url-encoded and exploded values for json
+query parameters. Please note that this style does not preserve the encoding of
+primitive types like numbers and booleans. They are always parsed as strings.
 
-For example:
+To filter results from the GET '/todo-list' endpoint in the
+todo-list example with a relation, { "include": [ { "relation": "todo" } ] },
+the following url-encoded query parameter can be used,
 
 ```
-GET /todos?filter[where][completed]=false
-// filter={where: {completed: 'false'}}
+   http://localhost:3000/todos?filter=%7B%22include%22%3A%5B%7B%22relation%22%3A%22todoList%22%7D%5D%7D
 ```
 
-As an extension to the deep-object encoding described by OpenAPI, when the
-parameter is specified with `style: "deepObject"`, we allow clients to provide
-the object value as a JSON-encoded string too.
+As an extension to the url-encoded style, LoopBack also supports queries with
+exploded values for json query parameters.
 
 For example:
 
 ```
-GET /todos?filter={"where":{"completed":false}}
-// filter={where: {completed: false}}
+GET /todos?filter[where][completed]=false
+// filter={where: {completed: 'false'}}
 ```
 
+LoopBack has switched the definition of json query params from the `exploded`,
```

Comment:
nitpick: how about move this sentence to the beginning of the section? 

And I remember there is a breaking change of the syntax, can you explain more about how to upgrade after the release?

Reply:
sorry never mind, found the change log in https://github.com/strongloop/loopback-next/blob/master/packages/rest/CHANGELOG.md#breaking-changes

Reply:
+1 to moving this sentence up

Reply:
agreed